### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

File: feedthefox/base/static/css/app.css [hunk 1]
```diff
@@ -290,6 +269,12 @@ h3 {
     }
 }
 
+@media only screen and (max-width:992px) {
```
Comment (akatsoulas) [inline feedthefox/base/static/css/app.css:272]: Personally, I would prefer a smaller size to hide the tabzilla link.

**Comment by akatsoulas** [comment]
Tested this locally and it works great. r+wc :coffee:

File: feedthefox/base/static/css/app.css (hunk 1 above)
Comment (comzeradd) [inline feedthefox/base/static/css/app.css:272]: Me too, but it overlaps with our many menu items. I'll merge it for now and revisit later.

**Pull request merged**